Assets/Mirror/Core/NetworkServer.cs

```diff
@@ -637,43 +638,64 @@ public static void SendToReadyObservers<T>(NetworkIdentity identity, T message,
         // called by transport
         static void OnTransportConnected(int connectionId)
```

Comment:
Probably better if this just called `OnTransportConnectedWithAddress(connectId, Transport.active.ServerGetClientAddress(connectionId))` the perf overhead of getting the address if we do reject the connection is probably neglible.

Reply:
Done